run/types/DeviceWrapper.ts

```
@@ -1095,29 +1099,42 @@ export class DeviceWrapper {
     }
   }
 
+  // TODO bring in iOS changes from QA-1265
   public async pushMediaToDevice(mediaFileName: string, forcedDate?: string) {
+    const filePath = path.join('run', 'test', 'specs', 'media', mediaFileName);
     if (this.isIOS()) {
       await runScriptAndLog(`touch -a -m -t ${forcedDate} 'run/test/specs/media/${mediaFileName}'`);
```

this was already the case before, but we are not checking that `forcedDate` is set to anything here. Should we? (and throw if not)?

---

well I guess this calls fails silently currently, because it would fail with `undefined`
`      await runScriptAndLog(`touch -a -m -t ${forcedDate} 'run/test/specs/media/${mediaFileName}'`);`

---

forcedDate is only in the iOS block, so it should always be there for iOS. I can make a note to look at it another time. This doesn't have to do with Miki's PR.

---

Added a TODO to force this for iOS